### PR TITLE
Screen shots for challenges

### PR DESCRIPTION
**Pull request by codenamejason**

### CODE REVIEW

**Comment by codenamejason** [comment]
screens for challenge 1 and 2

**Comment by codenamejason** [comment]
Wait to merge until I add challenge 0 @carletex

**Comment by carletex** [comment]
yo @codenamejason 

Thanks for this. I'll merge it once you add challenge 0.

**Comment by codenamejason** [comment]
@carletex should be good to go on the screens. Let me know if you see anything I may have missed and I will update.

File: packages/react-app/src/data/challenges/buyer-mints-nft.md [hunk 1]
```diff
@@ -1,187 +1,209 @@
-# 🏗 scaffold-eth - 🎫 Buyer Mints NFT
+# 🏗 scaffold-eth | 🏰 BuidlGuidl
 
-> (Counterfactual NFT minting example...)
+## 🚩 Challenge 0: 🎟 Simple NFT Example 🤓
```
Comment (carletex) [inline packages/react-app/src/data/challenges/buyer-mints-nft.md:3]: @codenamejason I think this is not challenge 0. `simple-nft-example` is.

File: packages/react-app/src/data/challenges/buyer-mints-nft.md [hunk 2]
```diff
@@ -1,187 +1,209 @@
...
 
-Deployer pays around (0.283719 ETH ~$500 at todays gas and price) for the initial contract but then NFTs are only minted once a buyer wants them. (The buyer of the NFT pays the gas to mint. ~$55)
+🎫 Create a simple NFT to learn basics of 🏗 scaffold-eth. You'll use [👷‍♀️ HardHat](https://hardhat.org/getting-started/) to compile and deploy smart contracts. Then, you'll use a template React app full of important Ethereum components and hooks. Finally, you'll deploy an NFT to a public network to share with friends! 🚀
 
-# 🏃‍♀️ Quick Start
+🏆 The final deliverable is an app that lets users purchase and transfer NFTs. Deploy your contracts to Rinkeby and then build and upload your app to a public web server. Share the url in the Challenge 0 telegram channel!!!
```
Comment (carletex) [inline packages/react-app/src/data/challenges/buyer-mints-nft.md:7]: @codenamejason I think we should remove the "telegram share" part

File: README.md [hunk 3]
```diff
@@ -57,36 +60,63 @@ Then re-run:
...
+
+![Screen Shot 2021-11-20 at 9 29 00 AM](https://user-images.githubusercontent.com/9419140/142730187-75941ef8-15b3-406d-ab6b-e175c21e522f.png)
+
+![Screen Shot 2021-11-20 at 9 37 36 AM](https://user-images.githubusercontent.com/9419140/142730302-63a60016-b7cb-4e69-a39b-93aa7a24f30e.png)
+
+You will now see your home page after signing a message.
+
+![Screen Shot 2021-11-20 at 9 36 31 AM](https://user-images.githubusercontent.com/9419140/142730252-a45af7af-98fd-4fd8-b639-4c478743163e.png)
+
+You can view all buidler by selecting "Buidlers"
```
Comment (carletex) [inline README.md:85]: @codenamejason Newly created users can only see "Portfolio" in their menu. Builders / Builds / Review are only for builders / admins.

We might want to explain the roles on a earlier section showing this screens (anon, registered, builder, admin). I can do that :D

**Review by carletex — COMMENTED** [review]
Hey @codenamejason 

I left some comments/issues, but I can tackle them if you are busy. Thanks!!

**Pull request closed without merging**